### PR TITLE
README.md: Initial install instructions, updated naming (sonic-*)

### PR DESCRIPTION
**Pull request by johnarnold**
Signed-Off-By: John Arnold <johnar@microsoft.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,39 @@
-[![Build Status](https://sonic-jenkins.westus.cloudapp.azure.com/buildStatus/icon?job=swss-build)](https://sonic-jenkins.westus.cloudapp.azure.com/job/swss-build)
-# Switch State Service - swss
+[![Build Status](https://sonic-jenkins.westus.cloudapp.azure.com/buildStatus/icon?job=sonic-swss-build)](https://sonic-jenkins.westus.cloudapp.azure.com/job/sonic-swss-build)
+# Sonic - Switch State Service - SwSS
```
Comment (stcheng) [inline README.md]: Sonic -> SONiC

File: README.md [hunk 2]
```diff
@@ -1,14 +1,42 @@
-[![Build Status](https://sonic-jenkins.westus.cloudapp.azure.com/buildStatus/icon?job=swss-build)](https://sonic-jenkins.westus.cloudapp.azure.com/job/swss-build)
-# Switch State Service - swss
-
-### This software repository must be treated as private and is only shared under NDA terms with Microsoft until publicly released.
+[![Build Status](https://sonic-jenkins.westus.cloudapp.azure.com/buildStatus/icon?job=sonic-swss-build)](https://sonic-jenkins.westus.cloudapp.azure.com/job/sonic-swss-build)
+# Sonic - Switch State Service - SwSS
```
Comment (stcheng) [inline README.md]: Sonic -> SONiC

**Comment by johnarnold** [comment]
Updated SONiC spelling and fixed the URL.

**Comment by stcheng** [comment]
thanks.

**Pull request closed without merging**